### ISSUE TITLE
Atomizer is order-dependent in the pipeline

### ISSUE THREAD

**Issue by rtyler** [awestruct/awestruct#505]
This could just be a documentation fix, but I've inadvertently discovered that this pipeline doesn't quite do what I would expect:

```ruby
Awestruct::Extensions::Pipeline.new do
 extension Awestruct::Extensions::Atomizer.new(:posts, '/news.atom', :feed_title=>'Awesome Blog Inc.')
 extension Awestruct::Extensions::Posts.new( '/blog', :posts )
end
```

```
% curl http://localhost:4242/news.atom
<?xml version="1.0" encoding="utf-8" ?>
<feed xml:lang="en-US" xmlns="http://www.w3.org/2005/Atom">
  <id>http://localhost:4242/</id>
  <title>Awesome Blog Inc.</title>
</feed>
```

If I change the order however, things work as expect and my blog posts are registered in the feed

**Comment by LightGuard** [comment]
Yes it is order dependent. Please feel free to fix on the site github repo.

On Tue, Jan 5, 2016 at 12:15 PM, R. Tyler Croy <notifications@github.com>
wrote:

> This could just be a documentation fix, but I've inadvertently discovered
> that this pipeline doesn't quite do what I would expect:
>
> Awestruct::Extensions::Pipeline.new do
>  extension Awestruct::Extensions::Atomizer.new(:posts, '/news.atom', :feed_title=>'Awesome Blog Inc.')
>  extension Awestruct::Extensions::Posts.new( '/blog', :posts )end
>
> % curl http://localhost:4242/news.atom
> <?xml version="1.0" encoding="utf-8" ?>
> <feed xml:lang="en-US" xmlns="http://www.w3.org/2005/Atom">
>   <id>http://localhost:4242/</id>
>   <title>Awesome Blog Inc.</title>
> </feed>
>
> If I change the order however, things work as expect and my blog posts are
> registered in the feed
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/awestruct/awestruct/issues/505>.
>



-- 
Jason Porter
http://en.gravatar.com/lightguardjp
